cfnstack/provisioner.go

```diff
@@ -231,3 +231,67 @@ func (c *Destroyer) Destroy() error {
 	_, err := cfSvc.DeleteStack(dreq)
 	return err
 }
+
+func (c *Provisioner) StreamCloudFormationNested(q chan struct{}, f *cloudformation.CloudFormation, stackId string, headStackName string, t time.Time) error {
```

Comment:
Would you mind commenting this func so that we can understand the scope at a glance?
For example, `StreamCloudFormationNested` streams all the events from the root, the contorl-plane, and worker node pool stacks`